src/utils/FileSystemUtils.ts

```diff
@@ -58,6 +58,9 @@ export class FileSystemUtils{
     }
 
     public static parseWorkDir(workdir: string): string {
+        if (!workdir) {
```

Comment:
I think that could be considered a default value for the parameter. 
The current approach works (but I don't think overriding a function parameter is 100% clean). Here is a different way. See if you like it a bit more: 
 

> function getDefault(): string {
>   // compute or retrieve the default value
>   return "default value";
> }
> 
> function yourFunction(input: string = getDefault()): void {
>   console.log(input);
> }